From e3f8846f06a9bd3aa43a1662ab8e682e8866ea13 Mon Sep 17 00:00:00 2001
From: Evangelos Foutras <evangelos@foutrelis.com>
Date: Thu, 26 Feb 2015 14:25:31 +0200
Subject: [PATCH] Drop use of the -remote option in Firefox helpers

The -remote option was removed in Firefox 36.

Affected helpers: Firefox, Iceweasel, IceCat.

The Mozilla Browser and Netscape Navigator helpers are left alone
since the respective browsers are no longer actively developed.

https://bugzilla.xfce.org/show_bug.cgi?id=11601
---
 exo-helper/helpers/firefox.desktop.in.in   | 4 ++--
 exo-helper/helpers/icecat.desktop.in.in    | 4 ++--
 exo-helper/helpers/iceweasel.desktop.in.in | 4 ++--
 3 files changed, 6 insertions(+), 6 deletions(-)

diff --git a/exo-helper/helpers/firefox.desktop.in.in b/exo-helper/helpers/firefox.desktop.in.in
index 9ed8104..c235a69 100644
--- a/exo-helper/helpers/firefox.desktop.in.in
+++ b/exo-helper/helpers/firefox.desktop.in.in
@@ -6,5 +6,5 @@ _Name=Mozilla Firefox
 StartupNotify=false
 X-XFCE-Binaries=firefox;firefox-gtk2;firefox-gtk;mozilla-firefox;
 X-XFCE-Category=WebBrowser
-X-XFCE-Commands=%B -remote "openURL(about:blank,new-window)";%B;
-X-XFCE-CommandsWithParameter=%B -remote "openURL(%s)";%B "%s";
+X-XFCE-Commands=%B;
+X-XFCE-CommandsWithParameter=%B "%s";
diff --git a/exo-helper/helpers/icecat.desktop.in.in b/exo-helper/helpers/icecat.desktop.in.in
index cb635f5..5a15ff2 100644
--- a/exo-helper/helpers/icecat.desktop.in.in
+++ b/exo-helper/helpers/icecat.desktop.in.in
@@ -6,5 +6,5 @@ _Name=Icecat
 StartupNotify=false
 X-XFCE-Binaries=icecat;icecat-gtk2;icecat-gtk;
 X-XFCE-Category=WebBrowser
-X-XFCE-Commands=%B -remote "openURL(about:blank,new-window)";%B;
-X-XFCE-CommandsWithParameter=%B -remote "openURL(%s)";%B "%s";
+X-XFCE-Commands=%B;
+X-XFCE-CommandsWithParameter=%B "%s";
diff --git a/exo-helper/helpers/iceweasel.desktop.in.in b/exo-helper/helpers/iceweasel.desktop.in.in
index 6fb6be8..99d9eca 100644
--- a/exo-helper/helpers/iceweasel.desktop.in.in
+++ b/exo-helper/helpers/iceweasel.desktop.in.in
@@ -6,5 +6,5 @@ _Name=Iceweasel
 StartupNotify=false
 X-XFCE-Binaries=iceweasel;iceweasel-gtk2;iceweasel-gtk;
 X-XFCE-Category=WebBrowser
-X-XFCE-Commands=%B -remote "openURL(about:blank,new-window)";%B;
-X-XFCE-CommandsWithParameter=%B -remote "openURL(%s)";%B "%s";
+X-XFCE-Commands=%B;
+X-XFCE-CommandsWithParameter=%B "%s";
-- 
2.3.1

